### PR TITLE
Add example data structure for storing friends

### PR DESCRIPTION
**Pull request by umitozmen**
Commiters
 @iulianabledea
@fedeninno

answer to creating group[https://github.com/UCL-RITS/rse-classwork-2020/issues/8]

### CODE REVIEW

File: group.py [hunk 1]
```diff
@@ -2,4 +2,21 @@
 
 # Your code to go here...
 
-my_group =
+my_group = {
+		'Jill' :{
+			'age': 26,
+			'job':['biologist'],
```
Comment (poppynikou) [inline group.py:8]: Good Idea in assuming one person can have multiple jobs i.e. using a list

File: group.py [hunk 2]
```diff
@@ -2,4 +2,21 @@
...
+			'relations':{'friend':['Zalika'],'partner':['John']}
```
Comment (poppynikou) [inline group.py:9]: Similarly, use of lists so a person can have multiple relationships

File: group.py [hunk 3]
```diff
@@ -2,4 +2,21 @@
...
+			},
+                'Zalika' :{
+			'age':28,
+			'job':['Artist'],
+			'relations':{'friend':['Jill']}
+			 }, 
+		'John' :{
+			'age':27,
+                        'job':['Writer'],
+                        'relations':{'partner':['Jill']
```
Comment (ucapgum) [inline group.py]: bracket not closed for inner dict, list could be helpful if more relations are added

Comment (poppynikou) [inline group.py]: You have added in reciprocal relationships between friends and partners,  what about other relationships?

**Review by markp121 — COMMENTED** [review]
Good that you have the functionality to add more than one job or relation for each person. Could think about adding some way to implement reciprocal relationships.

**Pull request closed without merging**